data/query.go

```diff
@@ -70,6 +101,58 @@ func ReviewQuery(ctx context.Context, cfg *config.Config, urlQuery url.Values, c
 	return validatedQueryParams, queryStringErr
 }
 
+// ReviewQuery ensures that all search parameter values given by the user are reviewed
+func ReviewDataAggregationQuery(ctx context.Context, cfg *config.Config, urlQuery url.Values, censusTopicCache *cache.Topic) (SearchURLParams, error) {
+	var validatedQueryParams SearchURLParams
+	validatedQueryParams.Query = urlQuery.Get("q")
+
+	fromDate, toDate, err := GetDates(context.TODO(), urlQuery)
+
+	if err == nil {
+		validatedQueryParams.AfterDate = fromDate
+		validatedQueryParams.BeforeDate = toDate
+	}
```

Comment:
Instead of `context.TODO()` send the ctx and handle the err
```suggestion
	fromDate, toDate, err := GetDates(ctx, urlQuery)
	if err != nil {
	  // Handle error
	}
	validatedQueryParams.AfterDate = fromDate
	validatedQueryParams.BeforeDate = toDate
```